### PR TITLE
APIView - Connect to cosmos and storage using managed identity

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Changes to connect to Cosmos and Storage using managed identity instead of using connection strings.

### CODE REVIEW

**Review by benbp — APPROVED** [review]

File: src/dotnet/APIView/APIViewWeb/MiddleWare/UITestsMiddleWare.cs [hunk 1]
```diff
@@ -18,19 +20,20 @@ public UITestsMiddleWare(RequestDelegate next)
 
         public async Task InvokeAsync(HttpContext httpContext, IConfiguration config)
         {
-            var cosmosClient = new CosmosClient(config["Cosmos:ConnectionString"]);
-            var dataBaseResponse = await cosmosClient.CreateDatabaseIfNotExistsAsync("APIView");
+            var cosmosClient = new CosmosClient(config["CosmosEndpoint"], new DefaultAzureCredential());
```
Comment (weshaggard) [inline src/dotnet/APIView/APIViewWeb/MiddleWare/UITestsMiddleWare.cs:23]: Do you really want to use DefaultAzureCredential here? Do we expect to ever use anything other than MI? What is the auth story for local testing?

**Review by weshaggard — APPROVED** [review]
Question about DAC but otherwise looks reasonable.

File: src/dotnet/APIView/APIViewWeb/MiddleWare/UITestsMiddleWare.cs (hunk 1 above)
Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/MiddleWare/UITestsMiddleWare.cs:23]: Let me check. I think this is not required to use dummy test instance.

**Review by chidozieononiwu — APPROVED** [review]

File: src/dotnet/APIView/APIViewWeb/MiddleWare/UITestsMiddleWare.cs (hunk 1 above)
Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/MiddleWare/UITestsMiddleWare.cs:23]: I will submit a separate PR to remove this Will file an issue to track it.

**Pull request merged**